test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/TestableHtmlGenerator.cs

```diff
@@ -95,12 +95,13 @@ public override IHtmlContent GenerateAntiforgery(ViewContext viewContext)
             return tagBuilder;
         }
 
-        protected override IDictionary<string, object> GetValidationAttributes(
+        public override void AddValidationAttributes(
```

Comment:
Does this compile after the `DefaultHtmlGenerator` change to `protected` for this method?